lib/install/install.go

```diff
@@ -101,6 +97,32 @@ func (i *Installer) Stop(ctx context.Context) error {
 	return nil
 }
 
+// Execute executes the install operation using the configured engine.
+// Implements server.Executor
+func (i *Installer) Execute(req *installpb.ExecuteRequest, stream installpb.Agent_ExecuteServer) error {
+	if !i.submit(req) && !req.IsResume() {
+		return status.Error(codes.AlreadyExists, "operation is already active")
```

Comment:
Installer does not allow to run multiple plan steps while the agent does.
The agent's main operation is fetching and displaying progress events (as far as the client is concerned) and running remote commands (for the installer) - this means that only the main operation is resumable if the client drops off - from the architecture point of view. If the client reconnects, it needs to continue fetching and displaying events from the ongoing operation.